regexes/client/browsers.yml

```diff
@@ -5,6 +5,13 @@
 # @license http://www.gnu.org/licenses/lgpl.html LGPL v3 or later
 ###############
 
+# Perfect Browser (http://www.perfectbrowser.com/)
+- regex: 'Perfect%20Browser(?:-iPad)?/(\d+[\.\d]+)'
```

Comment:
please add conditions for these user-agents.

* Mozilla/5.0 (iPad; CPU OS 12_1_1 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/12.0 Perfect/22.4 Mobile/15E148 Safari/604.1
* Mozilla/5.0 (iPhone; CPU iPhone OS 12_0_1 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/12.0 Perfect/22.2.1 Mobile/15E148 Safari/604.1

* PerfectBrowserPro/25.5 CFNetwork/1197 Darwin/20.0.0  
 more info http://perfectbrowser.com/pro.html